inspirehep/modules/workflows/workflows/article.py

```diff
@@ -491,18 +487,13 @@
 
 
 PROCESS_HOLDINGPEN_MATCH_SUBMISSION = [
+    handle_matched_holdingpen_wfs,
     IF_ELSE(
         has_same_source('holdingpen_matches'),
-        # form should detect this double submission
-        ERROR_WITH_UNEXPECTED_WORKFLOW_PATH,
```

Comment:
could we keep this part? it should be very rare, but different submissions are not interchangeable in principle.